lesson13/js/main.js

```diff
@@ -0,0 +1,146 @@
+'use strict';
+//Modal
+const modalContainer = document.getElementById('js-modalContainer');
+const modalBg = document.createElement('div');
+const modalWindow = document.createElement('div');
+
+function createModalWindow() {
+	const modal = document.createDocumentFragment();
+	const createMenuBtn = document.createElement('button');
+	const createCloseBtn = document.createElement('button');
+	const createMenuContainer = document.createElement('div');
+
+	//modal container
+	modalWindow.classList.add('bl_modal');
+	modalWindow.setAttribute('id', 'js-modalContainer');
+
+	//menu button
+	createMenuBtn.setAttribute('id', 'js-loadmenuBtn');
+	createMenuBtn.setAttribute('class', 'btn');
+	createMenuBtn.textContent = `MENU`;
+
+	//close button
+	createCloseBtn.setAttribute('id', 'js-closeModal');
+	createCloseBtn.setAttribute('class', 'el_closeModalBtn');
+	createCloseBtn.textContent = '×';
+
+	createMenuContainer.setAttribute('id', 'js-menu_container');
+
+	modal.appendChild(createMenuBtn);
+	modal.appendChild(createCloseBtn);
+	modal.appendChild(createMenuContainer);
+	modalWindow.appendChild(modal);
+	document.body.appendChild(modalWindow);
+
+	const clickBtn = document.getElementById('js-loadmenuBtn');
+	clickBtn.addEventListener(
+		'click',
+		(event) => {
+			callMenuContents();
+			event.stopPropagation();
+		},
+		false
+	);
+
+	const closeModlBtn = document.getElementById('js-closeModal');
+
+	closeModlBtn.addEventListener(
+		'click',
+		() => {
+			removeInnerHTML(modalWindow);
+			removeInnerHTML(menuLists);
+			document.body.removeAttribute('style');
+			modalBg.remove();
+			modalWindow.remove();
+		},
+		false
+	);
```

Comment:
And this one, too.

Reply:
Thank you for review!
I was able to function the button that closes the modal window!!!

[18dbe61](https://github.com/nyattsu72/hanson-frontend-cource/pull/18/commits/18dbe61f20915fb1f8de00792383d9aa734c0f66)